lib/cli/scripts/generate-sb-packages-versions.js

```diff
@@ -32,9 +33,13 @@ const run = async () => {
     .sort((package1, package2) => package1.name.localeCompare(package2.name))
     .reduce((acc, { name, version }) => ({ ...acc, [name]: version }), {});
 
-  await writeJson(path.join(__dirname, '..', 'src', 'versions.json'), packageToVersionMap, {
-    spaces: 2,
-  });
+  await writeFile(
+    path.join(__dirname, '..', 'src', 'versions.ts'),
+    dedent`
+      // auto generated file, do not edit
+      export default ${JSON.stringify(packageToVersionMap, null, 2)}
+    `
+  );
 };
```

Comment:
This script now generates a `versions,ts` file, instead of a `version.json` file.

It's easy enough to do (as you can see).

The reason I did this, is because `*.json` files are not considered "source files", and so when removing `--copy-files` flag from babel, this file went missing.

By generating the `.ts` file instead this problem goes away, and we actually get auto-complete.
I could make it even more type-safe, but that seemed not worth the time.

Reply:
![Screenshot 2022-01-26 at 14 47 24](https://user-images.githubusercontent.com/3070389/151174427-11beb7e8-73e0-4a0c-89d0-4dd302057792.png)

Reply:
@shilman I can revert this if you'd like...

We should remove the script if we're not using it...